pwiz_aux/msrc/utility/vendor_api/Bruker/TimsData.cpp

```diff
@@ -601,6 +604,11 @@ double TimsDataImpl::oneOverK0ToCCS(double oneOverK0, double mz, int charge) con
     return tims_oneoverk0_to_ccs_for_mz(oneOverK0, charge, mz);
 }
 
+double TimsDataImpl::oneOverK0ToCCS(double oneOverK0, int charge) const

```

Comment:
This does not seem to be used anymore? So this whole overload should be removed along with largestPeakMz.